echopype/tests/calibrate/test_calibrate_ek80.py

```diff
@@ -151,34 +178,70 @@ def test_ek80_BB_range(ek80_cal_path, ek80_ext_path):
     assert np.allclose(pyel_vals, ep_vals)
 
 
-def test_ek80_BB_power_Sv(ek80_cal_path, ek80_ext_path):
-    ek80_raw_path = ek80_cal_path / "2018115-D20181213-T094600.raw"  # rx impedance / rx fs / tcvr type
-    ed = ep.open_raw(ek80_raw_path, sonar_model="EK80")
+@pytest.mark.parametrize(
+    ("raw_data_path,raw_file_name,pyecholab_data_path,pyecholab_file_path, dask_array"),
+    [
+        (
+            "ek80_cal_path",
+            "2018115-D20181213-T094600.raw",
+            "ek80_ext_path",
+            "pyecholab/pyel_BB_p_data.pickle",
+            False,
+        ),
+        (
+            "ek80_cal_path",
+            "2018115-D20181213-T094600.raw",
+            "ek80_ext_path",
+            "pyecholab/pyel_BB_p_data.pickle",
+            True,
+        ),
+    ],
+)
+def test_ek80_BB_power_from_complex(
+    raw_data_path,
+    raw_file_name,
+    pyecholab_data_path,
+    pyecholab_file_path,
+    dask_array,
+    request,
+):
+    raw_data_path = request.getfixturevalue(raw_data_path)
+    ek80_raw_path = raw_data_path / raw_file_name  # rx impedance / rx fs / tcvr type
+
+    if dask_array:
+        ed = ep.open_raw(ek80_raw_path, sonar_model="EK80", destination_path="swap")
```

Comment:
Due to changes in #1185, `destination_path="swap"`  should be changed to `use_swap=True`